homeassistant/components/reolink/strings.json

```diff
@@ -38,6 +42,9 @@
       "init": {
         "data": {
           "protocol": "Protocol"
+        },
+        "data_description": {
+          "protocol": "Streaming protocol to use for the camera entities. RTSP supports 4K streams (h265 encoding) while RTMP and FLV do not. FLV is the least demanding on the camera."
```

Comment:
This seems a bit long to be presented nicely?

Reply:
This is how it looks like:

![afbeelding](https://github.com/home-assistant/core/assets/14811189/5ce360a1-b888-4dca-833c-f3b5fc23ec85)

I think it looks fine actually....